sdk/cosmos/azure-cosmos-spark_3_2-12/src/main/scala/com/azure/cosmos/spark/CosmosRowConverter.scala

```diff
@@ -113,6 +108,13 @@ private[cosmos] class CosmosRowConverter(
         new GenericRowWithSchema(values.toArray, schema)
     }
 
+    def fromObjectNodeToRowV1(schema: StructType,
```

Comment:
I assume V1 is for ChangeFeed V1 - please add because in two weeks no-one can remember what V1 and V2 stand for otherwise

like fromObjectNodeToChangeFeedRowV1

Reply:
ack, will do.